test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+
+describe RecipesController do
+  describe 'index' do
+    it "can get the index path" do
+      VCR.use_cassette("recipes") do
+        get recipes_path, params: {message: "chicken"}
+        must_respond_with :success
+      end
+    end
+
+    it 'fails with nonsensical input' do
+      VCR.use_cassette("recipes") do
+        proc {get recipes_path, params: {food: "hello"} }. must_raise
+      end
```

Comment:
It's good that you have a test case for this input. I'm not sure that I agree with the strategy you've chosen for handling it - controllers should usually report errors to the user via HTTP status codes and `flash` messages, rather than raising exceptions. But that's a design decision, and this is still a good test.